pages/api/graphql/schema/types/ProjectType.ts

```diff
@@ -8,6 +8,7 @@ module.exports = gql`
     description: String
     client_name: String
     client_description: String
+    image_url: String
```

Comment:
@Ovifer13 pls update the project schema

Reply:
Sorry, totally forgot I had to update there too. Omw